ui/package.json

```diff
@@ -6,7 +6,10 @@
     "dev": "vite",
     "build": "vue-tsc --noEmit && vite build --base=/opennms/ui/",
     "serve": "vite preview",
-    "test": "uvu -r tsm tests"
+    "test": "uvu -r tsm tests",
+    "lint": "eslint --ignore-path ../.gitignore --ext .ts,.vue .",
```

Comment:
Do we want to enforce either lint or formatting via CI?

Reply:
I think I'll start with this manually, but yes at some point either via CI or perhaps a pre-commit hook with husky